source/connectors/kafka-sink-data-formats.txt

```diff
@@ -0,0 +1,346 @@
+.kafka-sink-data-formats
+
+=================================
+Kafka Sink Connector Data Formats
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Supported Data Formats
+----------------------
+
+The MongoDB Kafka Sink Connector ``converter`` setting specifies the
+deserialization method for data it reads from a topic. The converter
+can deserialize the following data formats:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 1 4
+
+   * - Format Name
+     - Description
+
+   * - Avro
+     - An `open source serialization system <https://avro.apache.org>`_
+       that provides a compact binary format and a JSON-like API.
+       Integrates with the
+       `Confluent Schema Registry <https://www.confluent.io/confluent-schema-registry>`_
+       to manage schema definitions.
+
+   * - JSON with Schema
+     - JSON record structure with explicit schema information to
+       ensure the data matches the expected format.
+
+   * - JSON (plain)
+     - JSON record structure without an attached schema.
+
+   * - RAW JSON
+     - Serialized as a String. The JSON structure is not managed by
+       Kafka Connect.
+
+       .. note::
+
+          This format requires that the fields contain valid JSON even
+          when specifying the `StringConverter
+          <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/storage/StringConverter.html>`_
+
+.. note::
+
+   **RAW JSON** mode
+   requires the Strings to be valid JSON
```

Comment:
Thanks for spotting this. I've reworded this per one of Nathan's unrelated comments.

Since I'm mentioning a named setting, "RAW JSON", I think it is ok to then refer to JSON format in the same sentence. Were you concerned that I was defining a type of JSON using the term JSON in the explanation?

Reply:
I like the new phrasing! My concern, at the time, wasn't with defining a type of JSON using the word JSON in the explanation. It was actually with the repetitiveness of the sentence. But even though the new sentence is actually a bit more repetitive than the previous rendition, I actually think it reads well and is easy to understand.

Reply:
I'm still not sure what part is repetitive. Was there mention of the note elsewhere? What part of it is more repetitive than before?